scripts/env.d/vars

```diff
@@ -2,7 +2,7 @@ export DEVENVFLAVOR=""
 export DEVENVPREFIX="/dev/null"
 export DEVENVPATH_BACKUP="${PATH}"
 export DEVENVDLROOT="${DEVENVROOT}/var/downloaded"
-export DEVENVFLAVORROOT="${DEVENVFLAVORROOT:-$DEVENVROOT/flavors}"
+export DEVENVFLAVORROOT="$DEVENVROOT/flavors"
```

Comment:
This change removes the capability of manually setting the `DEVENVFLAVORROOT` from outside.  Can we the regression?

Reply:
Maybe the issue can be resolved in another way: unsetting `DEVENVFLAVORROOT` before changing flavor?